cfme/tests/cloud/test_instance_power_control.py

```diff
@@ -555,13 +539,19 @@ def test_stop(self, provider, testing_instance, ensure_vm_running,
             caseimportance: high
             initialEstimate: 1/4h
         """
-        testing_instance.wait_for_instance_state_change(desired_state=testing_instance.STATE_ON)
+        testing_instance.wait_for_power_state_change_rest(desired_state=testing_instance.STATE_ON)
```

Comment:
could you make such methods to use sentaku instead ? 
I don't insist to make such changes in this PR.

Reply:
Added a TODO for this.